src/tests/system/tests/test_authentication.py

```diff
@@ -210,7 +212,7 @@ def test_authentication__user_login_when_the_provider_is_offline(
     correct = "Secret123"
     wrong = "Wrong123"
     provider.user(user).add(password=correct)
-
+    client.sssd.sssd["services"] = "nss, pam, ssh"
```

Comment:
Please add this in `:setup:`

Reply:
dropping this from current PR.  This change is not intended for this PR.